components/detail/button/BtnFooter.style.tsx

```diff
@@ -45,4 +45,48 @@ const TimerCount = styled.p`
   font-style: normal;
 `;
 
-export { BtnWrapper, FilterBlur, StockTimer, TimerCount };
+const UpDownBlockWrapper = styled.div<{ status: string }>`
+  position: relative;
+  margin-top: 50px;
+  margin-bottom: 20px;
+  border-radius: 15px;
+  background: ${(props) =>
+    props.status == 'JOINED'
+      ? props.theme.colors.chart.point_up_100
+      : props.theme.colors.chart.point_down_100};
+  padding-top: 7px;
+  width: 86vw;
+  max-width: 475px;
+  height: 56px;
+  vertical-align: middle;
+  text-align: center;
+  line-height: 2vh;
+`;
+
+const UpDownBlockTitle = styled.div`
+  position: flex;
+  color: #fff;
```

Comment:
```suggestion
  color: ${(props) => props.theme.colors.basic.white}
```

같은 느낌으로 기본 색깔 사용하고 있긴 한대 나중에 해도 될 것 같슴당!

Reply:
새벽코딩해서 정신이 읎었네요 ㄷ ㄷ 바꿔두겟숨당